tests/e2e/tests/regression/insights/live-recommendations.e2e.ts

```diff
@@ -186,3 +182,18 @@ test
         // Verify that duplicates are not displayed
         await t.expect(redisVersionRecomSelector.count).eql(1, `${redisVersionRecom} recommendation duplicated`);
     });
+test.only('Verify that if user clicks on the database analysis button, new message appears  all DB', async t => {
```

Comment:
.only

Reply:
and please check the test name, it looks like not correct 'Verify that if user clicks on the database analysis button, new message appears all DB'